### PR TITLE
Ccn 33 productSearch

### PR DESCRIPTION
**Pull request by cramirez451**

### CODE REVIEW

File: base/templates/base.top.tpl.html [hunk 1]
```diff
@@ -14,6 +14,8 @@
         <!-- Collect the nav links, forms, and other content for toggling -->
         <div class="collapse navbar-collapse" id="BASE_TOP_COLLAPSE">
             <ul class="nav navbar-nav navbar-right">
+                <!-- TODO: sloppily put this in here for developing, needs to be formatted correctly -->
+                <li style="width:400px"><a><ordercloud-product-search></ordercloud-product-search></a></li>
```
Comment (spencerwalker) [inline base/templates/base.top.tpl.html]: remove the surrounding <li> and <a> elements and move the directive directly under div.collapse (line 15)

File: productSearch/templates/productSearch.component.tpl.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<form name="OrderCloudProductSearch">
```
Comment (spencerwalker) [inline productSearch/templates/productSearch.component.tpl.html]: add class="navbar-form and navbar-left" to the form element

File: base/templates/base.top.tpl.html (hunk 1 above)
Comment (robertsoniv) [inline base/templates/base.top.tpl.html]: I have a refactored base top in the oc-navbar branch that I'll work his search input into

File: productSearch/templates/productSearch.results.tpl.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- Product List -->
+        <div class="row">
+            <div class="col-md-4 col-lg-3" ng-repeat="product in productSearch.list.Items">
+                <div class="product-item" ui-sref="productDetail({productid: product.ID})">
+                    <h3 class="item-name">{{product.Name || product.ID}}</h3>
+                    <p class="item-id" ng-if="product.Name">{{product.ID}}</p>
```
Comment (robertsoniv) [inline productSearch/templates/productSearch.results.tpl.html]: @spencerwalker if you like the product list design on Alexa's product browse branch we should go with that here too

**Pull request merged**